src/components/YourCompany/BusinessContactInfo.vue

```diff
@@ -240,6 +241,10 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
     return this.businessContact.email !== this.originalBusinessContact.email
   }
 
+  private get isCorrection (): boolean {
+    return this.$route.name === FilingTypes.CORRECTION
```

Comment:
Incorrect type. Should be `RouteNames.CORRECTION`.

Actually, please move this getter to a method in "common-mixing.ts". (I have a similar need for alterations and that's what I'm doing.)

Reply:
![image](https://user-images.githubusercontent.com/10002889/96505286-c2aca780-120a-11eb-83a6-5ec85fef8283.png)

Reply:
Ah, will update!

Reply:
Awesome, thanks!